packages/react-composites/src/composites/CallComposite/components/CallArrangement.tsx

```diff
@@ -167,36 +168,38 @@ export const CallArrangement = (props: CallArrangementProps): JSX.Element => {
             )}
             {canUnmute && !!props.mutedNotificationProps && <MutedNotification {...props.mutedNotificationProps} />}
           </Stack.Item>
-          <Stack.Item grow style={callCompositeContainerFlex()}>
-            <Stack.Item styles={callGalleryStyles} grow>
-              {props.onRenderGalleryContent && (
-                <Stack verticalFill styles={mediaGalleryContainerStyles}>
-                  {props.onRenderGalleryContent()}
-                </Stack>
-              )}
-            </Stack.Item>
-          </Stack.Item>
-          {
+          {props.callControlProps?.options !== false &&
             /* @conditional-compile-remove(one-to-n-calling) @conditional-compile-remove(PSTN-calls) */
-            callPaneContent()
-          }
-        </Stack>
-        {props.callControlProps?.options !== false &&
-          /* @conditional-compile-remove(one-to-n-calling) @conditional-compile-remove(PSTN-calls) */
-          !isMobileWithActivePane && (
-            <Stack.Item className={callControlsContainerStyles}>
-              <CallControls
-                {...props.callControlProps}
-                containerWidth={containerWidth}
-                containerHeight={containerHeight}
-                isMobile={props.mobileView}
-                /* @conditional-compile-remove(one-to-n-calling) */
-                peopleButtonChecked={activePane === 'people'}
-                /* @conditional-compile-remove(one-to-n-calling) */
-                onPeopleButtonClicked={togglePeoplePane}
-              />
+            !isMobileWithActivePane && (
+              <Stack.Item className={callControlsContainerStyles}>
+                <CallControls
+                  {...props.callControlProps}
+                  containerWidth={containerWidth}
+                  containerHeight={containerHeight}
+                  isMobile={props.mobileView}
+                  /* @conditional-compile-remove(one-to-n-calling) */
+                  peopleButtonChecked={activePane === 'people'}
+                  /* @conditional-compile-remove(one-to-n-calling) */
+                  onPeopleButtonClicked={togglePeoplePane}
+                />
+              </Stack.Item>
+            )}
+          <Stack horizontal grow>
```

Comment:
Grouped the Gallery with the CallPane to properly display the CallPane on the screen. 
This is the primary fix for the CallPane issue Nan found.